Source/SwiftLintFramework/Rules/RuleConfigurations/TestCaseAccessibilityConfiguration.swift

```diff
@@ -1,10 +1,12 @@
 public struct TestCaseAccessibilityConfiguration: RuleConfiguration, Equatable {
     public private(set) var severityConfiguration = SeverityConfiguration(.warning)
     public private(set) var allowedPrefixes: Set<String> = []
+    public private(set) var xctestcaseSubclasses: Set<String> = []
 
     public var consoleDescription: String {
         return severityConfiguration.consoleDescription +
-            ", allowed_prefixes: [\(allowedPrefixes)]"
+            ", allowed_prefixes: [\(allowedPrefixes)]" +
+            ", xctestcase_subclasses: [\(xctestcaseSubclasses)]"
```

Comment:
To be not too specific I would like the option name `test_parent_classes` more. Perhaps there are other super classes that make a class a test case (in whichever sense) for which the rule is also reasonable.

If you agree, please also update the name of the class field and the CHANGELOG.

Reply:
How about `test_classes`, as they are not really "parent" classes per se?

Reply:
Doh - it's late in my timezone :-) `test_parent_classes` it is ...